services/ui-src/src/components/reports/DrawerReportPage.tsx

```diff
@@ -381,32 +390,59 @@ export const DrawerReportPage = ({ route, validateOnRender }: Props) => {
         </Box>
       )}
       <Box>
-        <Heading as="h3" sx={dashboardTitleStyling(canAddEntities)}>
-          {parseCustomHtml(verbiage.dashboardTitle)}
-        </Heading>
-        {isMcparReport && reportingOnIlos && !hasPlans && !hasIlos ? (
-          // if there are no plans and no ILOS added, display this message
-          <Box sx={sx.missingEntityMessage}>
-            {parseCustomHtml(verbiage.missingPlansAndIlosMessage || "")}
-          </Box>
-        ) : (isMcparReport && !reportingOnIlos && !hasPlans) ||
-          !entities?.length ? (
-          // if not reporting on ILOS, but missing entities, display this message
-          <Box sx={sx.missingEntityMessage}>
-            {parseCustomHtml(verbiage.missingEntityMessage || "")}
+        {reportingOnStandards ? (
+          // table of standards
+          <Box>
+            <Button
+              sx={sx.bottomAddEntityButton}
+              leftIcon={
+                <Image sx={sx.buttonIcons} src={addIconWhite} alt="Add" />
+              }
+              onClick={() => openRowDrawer()}
+            >
+              {verbiage.addEntityButtonText}
+            </Button>
+            {/* standards table */}
+            <Button
+              sx={sx.bottomAddEntityButton}
+              leftIcon={
+                <Image sx={sx.buttonIcons} src={addIconWhite} alt="Add" />
+              }
+              onClick={() => openRowDrawer()}
+            >
+              {verbiage.addEntityButtonText}
+            </Button>
```

Comment:
would it make sense to only show this button if there are entities in the table?

Reply:
In this case, no, we need this button to show up because the entity is being created through the Drawer, not a modal.